lib/src/generated_bindings.dart

```diff
@@ -0,0 +1,1432 @@
+// AUTO GENERATED FILE, DO NOT EDIT.
```

Comment:
We may need to add the cronet license header here. Since this file has been generated from third party code.

See this https://github.com/dart-lang/ffigen/blob/19da55249ba2efacc147982107698dcda04ea3a6/example/libclang-example/pubspec.yaml#L86-L92